azkaban-common/src/main/java/azkaban/utils/ExecuteAsUserUtils.java

```diff
@@ -0,0 +1,42 @@
+package azkaban.utils;
+
+import java.util.Arrays;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class is used as utility class for ExecuteAsUser script.
+ */
+public class ExecuteAsUserUtils {
+
+  private static final Logger logger = LoggerFactory.getLogger(ExecuteAsUserUtils.class);
+  private static final String ADD_GROUP_COMMAND = "/usr/sbin/groupadd";
```

Comment:
Should we rename this to GROUP_PATH?

Reply:
Good point. Changed it to ADD_GROUP_PATH and ADD_USER_PATH